README.md

```diff
@@ -166,6 +166,12 @@ and do the work of converting between user entities and documents to be indexed.
 * `reports`: Module built last, producing reports related to test coverage in particular.
 * `util`: Various modules containing util classes, both for runtime and for tests.
 
+## Showcase Project
+
+The project [Hibernate Search with Hibernate ORM Showcase](hibernate-search/tree/master/integrationtest/showcase/library) shows possible Hibernate Search application cases.
```

Comment:
I think this link is wrong, it does not work.
Maybe rather point to `integrationtest/showcase/library`?

Also, when I wrote the ticket I was merely thinking about adding a description of the showcase in the list above. Something like:

```
* `integrationtest`: Integration tests for backends (Elasticsearch, Lucene) and mappers (Hibernate ORM),
as well as any other technology Hibernate Search integrates with.
Here are some notable sub-directories:
  *  `performance`: performance tests.
  *  `showcase/library`: a sample application using Hibernate Search in a Spring Boot environment.
```

For users wanting to know more, the paragraph you added to the reference documentation should do the trick.